### DISCUSSION TITLE
Hello. The program does not show expenses for a specific tag

### DISCUSSION THREAD

**Discussion by nloshka404** [firefly-iii/firefly-iii#8001] [General]
Hello. The program does not show expenses for a specific tag. The situation is such that expenses with this label were entered 2 or 3 years ago and the program shows only for the current year and previous years are not taken into account. How to correctly display expenses with a specific label so that the display is done for 2 or 3 years ago? If you select an interval, everything shows the consumption with this label entered this year. and fixed dates for previous years are also displayed as empty values, but expenses can be seen by going to the all expenses tab and specifying the interval for the entire time and the required tags are set in expenses. Is this a bug in the program or was it intended to be this way? If this is an error, how can I fix it? thank you in advance!

**Comment by JC5** [comment]
On what page exactly are you missing these transactions?

**Reply by nloshka404** [reply]
label classification, required label interval all

**Reply by JC5** [reply]
Sorry, what's the url?

**Reply by nloshka404** [reply]
https://b.x512.ru/tags/show/98

**Reply by JC5** [reply]
Press "show all" at the bottom.

**Reply by nloshka404** [reply]
this only shows entries that were made this year

**Reply by JC5** [reply]
I can't replicate this. With old tags, even from 2019, the "/all" page shows all the expenses. Are there specific transactions you're missing?

**Reply by nloshka404** [reply]
I made a massive change of labels on all the necessary records. now displays labels. Perhaps the label field was filled in incorrectly and is now unrecognizable. Now everything shows how it is done and the total amount for the required label is displayed correctly. thanks, the problem went away after updating or rewriting the labels.

**Reply by JC5** [reply]
Glad to hear it, thanks for letting me know!
